elixir/apps/api/test/api/gateway/channel_test.exs

```diff
@@ -118,16 +118,15 @@ defmodule API.Gateway.ChannelTest do
                  %{protocol: :icmp}
                ]
              }
+
       assert payload.client == %{
                id: client.id,
                peer: %{
                  ipv4: client.ipv4,
                  ipv6: client.ipv6,
                  persistent_keepalive: 25,
-                 preshared_key: preshared_key,
```

Comment:
Hang on, why can we deserialize this without this field.

Reply:
This isn't going to work but our CI doesn't catch this because we never use `allow_access` I think?

Reply:
Yep, here we go: https://github.com/firezone/firezone/actions/runs/10569951962/job/29283741490?pr=6403#step:10:183